content/en/docs/components/training/mpi.md

```diff
@@ -21,7 +21,7 @@ cd mpi-operator
 kubectl create -f deploy/mpi-operator.yaml
 ```
 
-Alternatively, follow the [getting started guide](docs/started/getting-started/) to deploy Kubeflow.
+Alternatively, follow the [getting started guide](https://www.kubeflow.org/docs/started/getting-started/) to deploy Kubeflow.
```

Comment:
It's best to use a relative link rather than an absolute one, because absolute links break the doc versioning. I think the reason the link didn't work is a missing slash:

```
(/docs/started/getting-started/)
```

Reply:
Thanks @sarahmaddox ! 
I have modified the file to use relative link. Feel free to let me know if further change is needed.